docs/content/labs/IPv6/hostedcluster/hostedcluster.md

```diff
@@ -0,0 +1,297 @@
+In this section, we will focus on all the related objects necessary to achieve a Disconnected Hosted Cluster deployment.
+
+**Premises**:
+
+- HostedCluster Name: `hosted-ipv6`
+- HostedCluster Namespace: `clusters`
+- Disconnected: `true`
+- Network Stack: `ipv6`
+
+## Openshift Objects
+
+### Namespaces
+
+In a typical situation, the operator would be responsible for creating the HCP (HostedControlPlane) namespace. However, in this case, we want to include all the objects before the operator begins reconciliation over the HostedCluster object. This way, when the operator commences the reconciliation process, it will find all the objects in place.
+
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  creationTimestamp: null
+  name: clusters-hosted-ipv6
+spec: {}
+status: {}
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  creationTimestamp: null
+  name: clusters
+spec: {}
+status: {}
+```
+
+!!! note
+
+      We will **not** create objects one by one but will concatenate all of them in the same file and apply them with just one command.
+
+### ConfigMap and Secrets
+
+These are the ConfigMaps and Secrets that we will include in the HostedCluster deployment.
+
+```yaml
+---
+apiVersion: v1
+data:
+  ca-bundle.crt: |
+    -----BEGIN CERTIFICATE-----
+    -----END CERTIFICATE-----
+kind: ConfigMap
+metadata:
+  name: user-ca-bundle
+  namespace: clusters
+---
+apiVersion: v1
+data:
+  .dockerconfigjson: xxxxxxxxx
+kind: Secret
+metadata:
+  creationTimestamp: null
+  name: hosted-ipv6-pull-secret
+  namespace: clusters
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sshkey-cluster-hosted-ipv6
+  namespace: clusters
+stringData:
+  id_rsa.pub: ssh-rsa xxxxxxxxx
+---
+apiVersion: v1
+data:
+  key: nTPtVBEt03owkrKhIdmSW8jrWRxU57KO/fnZa8oaG0Y=
+kind: Secret
+metadata:
+  creationTimestamp: null
+  name: hosted-ipv6-etcd-encryption-key
+  namespace: clusters
+type: Opaque
+```
+
+### RBAC Roles
+
+While not mandatory, it allows us to have the Assisted Service Agents located in the same HostedControlPlane namespace as the HostedControlPlane and still be managed by CAPI.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: capi-provider-role
+  namespace: clusters-hosted-ipv6
+rules:
+- apiGroups:
+  - agent-install.openshift.io
+  resources:
+  - agents
+  verbs:
+  - '*'
+```
+
+### Hosted Cluster
+
+This is a sample of the HostedCluster Object
+
+!!! note
+
+    Please ensure you modify the appropriate fields to align with your laboratory environment.
+
+!!! warning
```

Comment:
@jparrill Does this warning apply only to the upstream docs, or should I include it in the official docs, too?